From 5e5149f5e8c7db9bd289c1ef45428a07a682d4a2 Mon Sep 17 00:00:00 2001
From: shucheng <shucheng@bd-apaas.com>
Date: Tue, 4 Jul 2023 23:02:10 +0800
Subject: [PATCH] backend fix path

---
 backend/main.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/backend/main.go b/backend/main.go
index 23a4af84..ecda73fa 100644
--- a/backend/main.go
+++ b/backend/main.go
@@ -50,9 +50,9 @@ func runServer(cfg *config.Config) {
 	)
 
 	handler := http.NewServeMux()
-	handler.HandleFunc("/api/", func(resp http.ResponseWriter, req *http.Request) {
+	handler.HandleFunc("/awecloud/hubble/api/", func(resp http.ResponseWriter, req *http.Request) {
 		// NOTE: GRPC server handles requests with URL like "ui.UI/functionName"
-		req.URL.Path = req.URL.Path[len("/api/"):]
+		req.URL.Path = req.URL.Path[len("/awecloud/hubble/api/"):]
 		wrappedGrpc.ServeHTTP(resp, req)
 	})
 
-- 
2.39.2

